configure.ac

```diff
@@ -2869,7 +2929,8 @@ AC_MSG_RESULT([CXX                : ${CXX}])
 AC_MSG_RESULT([CONFIGURE_CXXFLAGS : ${CONFIGURE_CXXFLAGS}])
 AC_MSG_RESULT([SPECIFIED_CXXFLAGS : ${SPECIFIED_CXXFLAGS}])
 AC_MSG_RESULT([EXTRA_CXXFLAGS     : ${EXTRA_CXXFLAGS}])
-AC_MSG_RESULT([LDFLAGS            : ${LDFLAGS}])
```

Comment:
To double check, this shouldn't affect existing use cases, since LDFLAGS is included in SPECIFIED_LDFLAGS, is that correct?

If so, maybe still include the final FDFLAGS here?

Reply:
Thanks for explaining @ilvokhin. Everything looks good. Merging.